features/bsm/service/impl/src/main/java/org/opennms/netmgt/bsm/service/internal/edge/ChildEdgeImpl.java

```diff
@@ -70,4 +70,9 @@ public String toString() {
                 .add("child", getChild() == null ? null : getChild().getId())
                 .toString();
     }
+
+    @Override
+    public String getFriendlyName() {
+        return null;
```

Comment:
> On Feb 26, 2016, at 4:59 AM, mvrueden <notifications@github.com> wrote:
> 
> Can we add a comment, that the friendly name of business services do not have a friendly name. Otherwise one may think it was forgotten to be implemented

Can we automatically generate/suggest one based on the reduction key?

:David

David Hustace
The OpenNMS Group, Inc.